### DISCUSSION TITLE
Variable Fraction Scale.

### DISCUSSION THREAD

**Discussion by davpro7** [mate-desktop/mate-desktop#607] [Ideas]
Hi!
I am prefering Mate if i could get the right size, so I am now running XFCE  where I can get the right size.
Before, I think there was no option 1600x900 res in Ubuntu.
I had a chat with some, and it was like.
-You should do scaling.
So, I have doing scaling in both Windows and XFCE-desktop around
0,9  (enlargement 10-15%).
But in Mate Desktop, there is no alternative. Is it hard or easy to implement an option for variable scaling like it is in XFCE desktop?
I would do it my self if I could. I've done some coding, but I am a newbie in this case. :-)

Regards
David

**Comment by lukefromdc** [comment]
This sort of thing can only be done in Xorg and GTK3 with Xrandr, and fractional scaling in Xrandr is quite heavy as to my 
knowledge it involves scaling first up and then back down. If you want fractional scaling that works now and have or can
build and install MATE 1.28 with wayland support enabled, try installing mate-wayland-session and running it from the
display manager (SDDM etc). Make sure to install wdisplays, and you will have a graphical application capable of setting
decimal scaling values. NOT sure how efficient fractional scaling is in wayfire, and I know development continues, but it
just might work for you because this is one of those things more easily done in wayland than in X11 at the moment.

**Reply by davpro7** [reply]
Wow. Thanx man for a very good and detailed answer!
It would be interesting to give it a try. :-)
